next-app/src/lib/server/types.d.ts

```diff
@@ -0,0 +1,31 @@
+import type { HttpMethod } from '@sveltejs/kit/types/private'
+
+type Rpc = {
```

Comment:
Some of the reading I was doing almost seemed like the type was a more appropriate place to start unless you needed the additional features of an `interface`.  If you need something a `type` doesn't provide, then you can increase complexity by moving to an `interface` but in that case it will be known why the `interface` was needed.